docs/ReactInstanceSettings-api-windows.md

```diff
@@ -21,16 +21,17 @@ Base path used for the location of the bundle.
  string `ByteCodeFileUri`
 
 Set this to a location the application has write access to in order for bytecode to be successfully cached. See [`EnableByteCodeCaching`](#enablebytecodecaching).
+**Note that currently the byte code generation is not implemented for UWP applications.**
 
 ### DebugBundlePath
  string `DebugBundlePath`
 
-When loading from a bundle server (such as metro), this is the path that will be requested from the server.  If this is not provided the value of [`JavaScriptBundleFile`](#javascriptbundlefile) or [`JavaScriptMainModuleName`](#javascriptmainmodulename) is used.
+When loading from a bundle server (such as metro), this is the path that will be requested from the server. If this is not provided the value of [`JavaScriptBundleFile`](#javascriptbundlefile) or [`JavaScriptMainModuleName`](#javascriptmainmodulename) is used.
 
 ### DebugHost
  string `DebugHost`
 
-> **Deprecated**: This has been replaced with [`SourceBundleHost`](#sourcebundlehost) and [`SourceBundlePort`](#sourcebundleport) and will be removed in a future version.
+> **Deprecated**: This has been replaced with [`SourceBundleHost`](#sourcebundlehost) and [`SourceBundlePort`](#sourcebundleport) and will be removed in a version 0.65.

```

Comment:
```suggestion
> **Deprecated**: This has been replaced with [`SourceBundleHost`](#sourcebundlehost) and [`SourceBundlePort`](#sourcebundleport) and will be removed in version 0.65.

```
Also... aren't these the doc updates for 0.65? Shouldn't this be removed now? Do we have tracking bugs for all of these?

Reply:
The content here reflects what is in the code repo master branch. The work to remove these APIs has not been done. I don't believe we have bugs tracking it but we should.

Reply:
Filed microsoft/react-native-windows#7486